src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
It's a good practice to include a fallback font in case the main font doesn't load. You can add a general font family like `sans-serif` as a fallback. For example: `font-family: Roboto, sans-serif;`. [Checklist: Remember to use fallback fonts]